### ISSUE TITLE
September data dump is missing

### ISSUE THREAD

**Issue by ahwagner** [griffithlab/civic-server#358]
Looks like our September database dump didn't run. We should queue up a dump and ensure that this will not be a recurring issue.

**Comment by susannasiebert** [comment]
I chatted with Adam about this on Slack. In order to create this data dump one would need to be able to ssh to civic.gsc.wustl.edu and then need sudo permissions in order to do the following:

**Comment by susannasiebert** [comment]
Note to self:
>
> 
> adam [2:41 PM] 
> as some background info, basically to debug it, in the console you can do
> 
> ```x = Delayed::Job.all
> ```
> 
> Which will give you objects for each of the scheduled tasks, those objects have failure counts, last error messages, and a handler (which is the actual task to run) so you can find the one for generate monthly tsvs and then poke around at the error message

**Comment by acoffman** [comment]
Data dump is generated for September! I did some initial debugging and couldn't find a smoking gun, everything looks as I would expect. But as this has happened a couple of times, clearly _something_ isn't right. Here's what I've done so far:

```ruby
//grab any scheduled jobs who have the handler GenerateMonthlyTSVs
jobs = Delayed::Job.all.select { |j| j.handler =~ /monthly/i}
jobs.size
=> 1
jobs.first.created_at
=> "2017-09-01 05:00:00"
jobs.first.last_error
=> nil
jobs.first.failed_at
=> nil
jobs.first.run_at
=> "2017-10-01 00:00:00"
```

So upon initial viewing it looks like this ran on Sept 1, didn't fail, and is scheduled to run again on October 1 as we'd expect. Nothing out of the ordinary there....

Its also a little weird because, looking at the implementation [here](https://github.com/griffithlab/civic-server/blob/master/app/jobs/generate_monthly_tsvs.rb#L4), it calls into the actual [GenerateTsvs](https://github.com/griffithlab/civic-server/blob/master/app/jobs/generate_tsvs.rb) parent class to create the files, and then reschedules itself afterwards. If the files failed to generate and some sort of exception was thrown, the task shouldn't get rescheduled and the failure should get logged in that table I was looking at. The job also ran correctly to completion when I started it manually from the console.

So, something here is odd and I'm not 100% sure what yet, but I'm sure it'll be something silly and obvious that I'm just not thinking of right now. I'll keep digging!
